### DISCUSSION TITLE
Datadog is not working

### DISCUSSION THREAD

**Discussion by MedhatNakhla** [GoogleCloudPlatform/terraformer#795] [Q&A]
ec2-user@ip-10-0-1-225 datadog-terraform]$ terraformer import datadog --resources=dashboard --filter=dashboard=SandboxHealth --api-key ********  --app-key **********
2021/02/16 12:16:14 unknown flag: --resources
[ec2-user@ip-10-0-1-225 datadog-terraform]$ 

However, I copied the datadog provider folder to /home/ec2-user/.terraform.d/plugins/linux_x86-64

Any idea ?

**Comment by sergeylanzman** [comment]
It's last version?

**Comment by fabibal** [comment]
I think you have download the wrong version of terraformer
